src/PlusDataCollection/Capistrano/vtkPlusCapistranoVideoSource.cxx

```diff
@@ -1141,11 +1141,8 @@ PlusStatus vtkPlusCapistranoVideoSource::InternalUpdate()
   GetObject(this->Internal->DataHandle, sizeof(BITMAP), &this->Internal->Bitmap);
 
   vtkPlusDataSource* aSource=NULL;
-  if( this->GetFirstActiveOutputVideoSource(aSource) != PLUS_SUCCESS )
-  {
-    LOG_ERROR("Unable to retrieve the video source in the ICCapturing device.");
-    return PLUS_FAIL;
-  }
+  ERROR_FAIL_IF(this->GetFirstActiveOutputVideoSource(aSource) != PLUS_SUCCESS,

```

Comment:
Just an FYI, this should never fail as you can guarantee its existence in NotifyConfigured()

Reply:
Why was it being checked then? I only re-wrote the check, I did not introduce it.

Reply:
The NotifyConfigured came in after a lot of these were written, and I did not go back and fix all instances as it's not a critical change. More it's just cleaning up a bit to only check once at post-config/pre-connect time for any configuration errors.

Again, optional, but good practice.